# Angles for dynamixel pan-tilt
# Angles in degrees, range -180 -> 180

float64 pan_angle
float64 tilt_angle 
